cnf-tests/testsuites/e2esuite/dpdk/dpdk.go

```diff
@@ -271,13 +277,15 @@ sleep INF
 				Expect(bytes).To(BeNumerically(">", 0))
 
 				By("Parsing output from the DPDK application")
-				Eventually(func() string {
+				Eventually(func() bool {
 					out, err = pods.GetLog(dpdkWorkloadPod)
 					Expect(err).ToNot(HaveOccurred())
-					return out
-				}, 8*time.Minute, 1*time.Second).Should(ContainSubstring("NIC statistics for port"),
-					"Cannot find accumulated statistics")
-				checkRxOnly(out)
+					if !strings.Contains(out, "NIC statistics for port") {
```

Comment:
is this necessary? Without it, checkRxOnly would perform the same check I guess

Reply:
you are right thanks! :)